e2e/cypress/integration/channel_sidebar/category_collapsing_spec.js

```diff
@@ -139,32 +139,38 @@ describe('Channel sidebar', () => {
     });
 
     it('should retain the collapsed state of categories when unread filter is enabled/disabled', () => {
-        // * Verify that CHANNELS starts expanded
-        cy.get('.SidebarChannelGroupHeader:contains(CHANNELS) i').should('not.have.class', 'icon-rotate-minus-90');
+        cy.getCurrentTeamId().then((teamId) => {
+            cy.apiCreateChannel(teamId, 'channel-test', 'Channel Test').then(({channel}) => {
+                cy.postMessageAs({sender: sysadmin, message: 'Test', channelId: channel.id});
 
-        // * Verify that all categories are visible
-        cy.get('.SidebarChannelGroupHeader:contains(CHANNELS)').should('be.visible');
-        cy.get('.SidebarChannelGroupHeader:contains(CHANNELS) i').should('be.visible').should('not.have.class', 'icon-rotate-minus-90');
-        cy.get('.SidebarChannelGroupHeader:contains(DIRECT MESSAGES)').should('be.visible');
-        cy.get('.SidebarChannelGroupHeader:contains(DIRECT MESSAGES) i').should('be.visible').should('not.have.class', 'icon-rotate-minus-90');
+                // * Verify that CHANNELS starts expanded
+                cy.get('.SidebarChannelGroupHeader:contains(CHANNELS) i').should('not.have.class', 'icon-rotate-minus-90');
```

Comment:
This ```cy.get('.SidebarChannelGroupHeader:contains``` is being used for 21 times in one file
can we make a local function with parameter